src/fragments/lib/geo/js/maps.mdx

```diff
@@ -160,6 +162,15 @@ map.setStyle("myAmplifyGeoEsriTopographicMap"); // map name received from getAva
 map.resize(); // forces the map to re-render
 ```
 
+## Removing a map from the DOM
+
+When it's time to remove the map from the DOM, you can use the `.remove` method of the generated map. This will clean up and release all resources associated with the map (DOM elements, event bindings, web workers, and WebGL resources).
+Note: Afterwards, you must not call any other methods on the map.
```

Comment:
Will something destructive happen if a method is called? Or, is it just that the method will not work? Just trying to gauge if this should be bolded or called out more as a `warning`.

Reply:
Yeah, I think it's important to call out that if you don't do this in a framework like React you'll have a memory leak

Reply:
> Will something destructive happen if a method is called? Or, is it just that the method will not work? Just trying to gauge if this should be bolded or called out more as a `warning`.

I'm actually not positive _what_ happens here, that's all the information MapLibre gives in their documentation about `remove()`, would need to do some manual messing around to know more atm.

Happy to update this more as a warning and add something specific calling out React for memory leaks.

https://maplibre.org/maplibre-gl-js-docs/api/map/#map#remove

![image](https://user-images.githubusercontent.com/16496746/140573165-00485501-448a-4403-b858-af6128350385.png)

Reply:
Added updates you can see below around this.